tests/settings_loader_unittest.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pickle
+import unittest
+
+
+class SettingsTest(unittest.TestCase):
+    def setUp(self):
+        self.TEST_SETTINGS_FILE = 'tests/settings_test.py'
+        if os.path.exists(self.TEST_SETTINGS_FILE):
+            raise RuntimeError(f"'{self.TEST_SETTINGS_FILE} already exists, would be deleted by running the tests.")
+
+        # Disable loading from settings file
+        os.environ['SIS_GLOBAL_SETTINGS_FILE'] = ''
+
+    def tearDown(self):
+        if os.path.exists(self.TEST_SETTINGS_FILE):
+            os.unlink(self.TEST_SETTINGS_FILE)
+
+    def test_load_settings_from_file(self):
+        import sisyphus.global_settings as gs
+
+        with open(self.TEST_SETTINGS_FILE, 'wt') as f:
+            f.write("""
+file_test1 = 'foo'
+file_test2 = 4
+def test_function():
+    return 42
+            """)
+
+        gs.update_global_settings_from_file(self.TEST_SETTINGS_FILE)
+        assert gs.file_test1 == 'foo'
+        assert gs.file_test2 == 4
+        assert gs.test_function() == 42
```

Comment:
Would
```suggestion
        with open(self.TEST_SETTINGS_FILE, 'wt') as f:
            f.write("""
file_test1 = 'foo'
file_test2 = 4
DEFAULT_ENVIRONMENT_SET['PATH'] += ':bar'
def test_function():
    return 42
            """)

        gs.update_global_settings_from_file(self.TEST_SETTINGS_FILE)
        assert gs.file_test1 == 'foo'
        assert gs.file_test2 == 4
        assert gs.DEFAULT_ENVIRONMENT_SET['PATH'][-4:] == ':bar'
        assert gs.test_function() == 42
```
now still work?

Reply:
Yes it still works, it's now part of the unittests.